examples/platform_channel/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -119,18 +122,26 @@
 			name = "Supporting Files";
 			sourceTree = "<group>";
 		};
+		CF3B75C9A7D2FA2A4C99F110 /* Frameworks */ = {
```

Comment:
Yeah - this is in keeping with the others. While it starts empty, it's also not an uncommon setup. We just as well could kill it off everywhere too.